service/iot-service-client/pom.xml

```diff
@@ -95,6 +91,12 @@
             <groupId>com.microsoft.azure</groupId>
             <artifactId>azure-storage</artifactId>
             <version>4.0.0</version>
+            <exclusions>
+                <exclusion>
```

Comment:
what is this exclusion for?

Reply:
this Azure Storage version comes with a dependency on the excluded library so that we can control the version that our library uses